### PR TITLE
Apply m3 to chat bubble

### PR DESCRIPTION
**Pull request by sherlockvn**
https://github.com/linagora/twake-on-matrix/assets/43041967/500b1318-681c-4fae-963c-584a99ac19bc

### CODE REVIEW

File: lib/pages/chat/chat_app_bar_title.dart [hunk 1]
```diff
@@ -44,79 +45,78 @@ class ChatAppBarTitle extends StatelessWidget {
...
+                          height: 15,
+                          decoration: BoxDecoration(
+                            border: Border.all(
+                              color: Colors.white,
+                              width: 2,
+                            ),
+                            color:
+                                room.directChatPresence?.currentlyActive == true
+                                    ? const Color(0xFF5AD439)
+                                    : const Color(0xFF818C99),
```
Comment (hoangdat) [inline lib/pages/chat/chat_app_bar_title.dart:83]: IMO, we should separate in ***_sytle file with the dedicated name

File: lib/pages/chat/chat_input_row.dart [hunk 2]
```diff
@@ -90,16 +91,12 @@ class ChatInputRow extends StatelessWidget {
                   duration: FluffyThemes.animationDuration,
                   curve: FluffyThemes.animationCurve,
                   height: 56,
-                  width: 56,
+                  width: 52,
```
Comment (hoangdat) [inline lib/pages/chat/chat_input_row.dart:94]: magic number, please style it

File: lib/pages/chat/chat_input_row.dart [hunk 3]
```diff
@@ -221,15 +215,14 @@ class ChatInputRow extends StatelessWidget {
                           focusNode: controller.inputFocus,
                           controller: controller.sendController,
                           decoration: InputDecoration(
-                            hintText: L10n.of(context)!.writeAMessage,
+                            hintText: "Chat message",
```
Comment (hoangdat) [inline lib/pages/chat/chat_input_row.dart:218]: dont forget to use l10n

File: assets/images/ic_send.svg
[the remark this answers is not in the document]
Comment (sherlockvn) [reply assets/images/ic_send.svg:1]: Yes. i can't find any icon for it, so download the new one from figma design

File: lib/pages/chat/events/message_reactions.dart [hunk 4]
```diff
@@ -46,9 +47,47 @@ class MessageReactions extends StatelessWidget {
...
+          child: Container(
+            width: 28.0,
+            height: 28.0,
+            decoration: BoxDecoration(
+              color: Theme.of(context).colorScheme.surface,
+              border: Border.all(color: Colors.white),
+              shape: BoxShape.circle,
+            ),
+            padding: const EdgeInsets.only(top: 1.0, bottom: 4.0),
+            child: const Icon(Icons.more_horiz_rounded, size: 20,),
```
Comment (imGok) [inline lib/pages/chat/events/message_reactions.dart:87]: Could you fix the indentation please

File: lib/pages/chat/events/message_reactions.dart [hunk 5]
```diff
@@ -154,18 +194,23 @@ class _Reaction extends StatelessWidget {
...
-          borderRadius: BorderRadius.circular(AppConfig.borderRadius),
+          border: Border.all(color: Colors.white),
+          borderRadius: BorderRadius.circular(16.0),
+        ),
+        padding: const EdgeInsets.symmetric(horizontal: 5.0, vertical: 3.0),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            content,
+            const SizedBox(width: 4,),
```
Comment (imGok) [inline lib/pages/chat/events/message_reactions.dart:208]: Indentation pls

File: lib/pages/chat/events/message_reactions.dart (hunk 4 above)
Comment (sherlockvn) [reply lib/pages/chat/events/message_reactions.dart]: I think we don't need it :)

**Comment by hoangdat** [comment]
Screenshot 2023-05-15 at 05 17 44

**Comment by hoangdat** [comment]
image

the padding is wrong. It should be `horizontal = 8, vertival = 8`

File: lib/config/themes.dart [hunk 6]
```diff
@@ -67,7 +69,12 @@ abstract class FluffyThemes {
             borderSide: BorderSide.none,
             borderRadius: BorderRadius.circular(AppConfig.borderRadius / 2),
           ),
-          filled: true,
+          hintStyle: TextStyle(
+            fontSize: 17,
+            fontWeight: FontWeight.w400,
+            color: LinagoraRefColors.material().neutralVariant[60],
+            overflow: TextOverflow.ellipsis
+          )
```
Comment (hoangdat) [inline lib/config/themes.dart:77]: why not use `m3/body/large`

File: lib/config/themes.dart [hunk 7]
```diff
@@ -200,6 +207,22 @@ abstract class FluffyThemes {
               ? LinagoraSysColors.material().primary
               : LinagoraSysColors.material().primaryDark,
         ),
+        iconButtonTheme: IconButtonThemeData(
+          style: ButtonStyle(
+            iconSize: MaterialStateProperty.all(iconSize),
+            iconColor: MaterialStateProperty.all(
+              brightness == Brightness.light
+                ? Colors.black
+                : Colors.white
```
Comment (hoangdat) [inline lib/config/themes.dart:216]: IMO, no black/white in design. why not use 
image

File: pubspec.yaml [hunk 8]
```diff
@@ -37,7 +37,7 @@ dependencies:
   flutter_localizations:
     sdk: flutter
   flutter_map: ^3.1.0
-  flutter_matrix_html: ^1.1.0
+#  flutter_matrix_html: ^1.1.0
```
Comment (hoangdat) [inline pubspec.yaml:40]: 😨

Comment (sherlockvn) [reply pubspec.yaml:40]: haha

**Review by hoangdat — APPROVED** [review]

**Pull request merged**